modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/CSharpClientCodegen.java

```diff
@@ -86,6 +86,18 @@ public CSharpClientCodegen() {
 
         cliOptions.clear();
 
+        typeMapping.put("boolean", "bool");
```

Comment:
I may move these type mapping to abstract C# code generator instead with another PR.